`npm run build` fails

Some .js files imports .css files, but actual suffixes are .scss.

```
% git clone https://github.com/KualiCo/nats-streaming-console.git
% cd nats-streaming-console
% npm install
...
% npm run build

> nats-streaming-console@1.0.1 build /home/hidai/nats/nats-streaming-console
> react-scripts build

Creating an optimized production build...
Failed to compile.

Module not found: Error: Can't resolve './index.css' in '/home/hidai/nats/nats-streaming-console/src'


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nats-streaming-console@1.0.1 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR!-
npm ERR! Failed at the nats-streaming-console@1.0.1 build script.
```

--- a/README.md
+++ b/README.md
@@ -39,6 +39,7 @@ If you just want to build and run it. That is pretty easy too. You will need git
 git clone https://github.com/KualiCo/nats-streaming-console.git
 cd nats-streaming-console
 npm install
+npm run build-css
 npm run build
 node server
 ```

